wqflask/base/trait.py

```diff
@@ -438,6 +507,18 @@ def retrieve_info(self, get_qtl_info=False):
                     self.location_repr = 'Chr%s: %.6f' % (self.chr, float(self.mb))
                     self.location_value = trait_location_value
 
+            elif self.dataset.type == "Geno":
+                self.location_repr = 'N/A'
+                self.location_value = 1000000
```

Comment:
Can we move all state variables into a separate object? The use of self is kinda dangerous and a code smell in my opinion.